<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValueSelector</class>
 <widget class="QWidget" name="ValueSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="variableName">
     <property name="text">
      <string>V</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDoubleSpinBox" name="variableMin">
     <property name="minimum">
      <double>-999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="variableMax">
     <property name="minimum">
      <double>-1000000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="variableValue">
     <property name="minimumSize">
      <size>
       <width>633</width>
       <height>0</height>
      </size>
     </property>
     <property name="minimum">
      <double>-1000000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QwtSlider" name="variableSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtSlider</class>
   <extends>QWidget</extends>
   <header>qwt_slider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
